packages/tooltip/src/lib/Tooltip.stories.tsx

```diff
@@ -0,0 +1,81 @@
+// Each exported component in the package should have its own stories file
+
+import type { StoryObj } from '@storybook/react';
+import { Tooltip, TooltipProps } from './Tooltip';
+
+// TODO: use our components when forwardRef is being applied correctly
```

Comment:
Can we get rid of this TODO?

Reply:
I think we should try to use our own components in our docs so I we should probably keep this.